parm/config/gfs/config.resources

```diff
@@ -294,13 +301,16 @@ case ${step} in
     export layout_y=${layout_y_atmanl}
 
     walltime="00:30:00"
-    ntasks_gdas=$(( layout_x * layout_y * 6 ))
-    ntasks_gfs=$(( layout_x * layout_y * 6 ))
-    threads_per_task_gdas=1
-    threads_per_task_gfs=${threads_per_task_gdas}
-    tasks_per_node_gdas=$(( max_tasks_per_node / threads_per_task_gdas ))
-    tasks_per_node_gfs=$(( max_tasks_per_node / threads_per_task_gfs ))
     memory="96GB"
+    if [[ "${RUN}" = *gdas ]]; then
+      ntasks=$(( layout_x * layout_y * 6 ))
+      threads_per_task=1
+      tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    elif [[ "${RUN}" = *gfs ]]; then
+      ntasks=$(( layout_x * layout_y * 6 ))
+      threads_per_task=1
+      tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    fi
```

Comment:
No need for an if-elif block here since the variables are all the same

```suggestion
    ntasks=$(( layout_x * layout_y * 6 ))
    threads_per_task=1
    tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
```